src/test/java/org/jitsi/meet/test/util/MeetUIUtils.java

```diff
@@ -1004,20 +999,7 @@ public static void waitForNinjaIcon(WebDriver driver)
     public static void waitForNinjaIcon(WebDriver driver, String endpointId)
     {
         TestUtils.waitForElementByXPath(driver,
-            "//span[@id='participant_" + endpointId + "']//span[contains(@class,'connection_ninja')]", 15);
-    }
-
-    /**
-     * Waits for dominant speaker icon to appear in remote video of a participant.
-     * @param driver the <tt>WebDriver</tt> instance where the check will be
-     * performed.
-     * @param endpointId the endpoint ID of the participant whose dominant speaker icon
-     * status will be checked.
-     */
-    public static void waitForDominantspeaker(WebDriver driver, String endpointId)
```

Comment:
Why this is removed without an alternative? Seems to me this changes the logic of the tests

Reply:
My bad. Fixed it :D